src/Passes/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+cmake_minimum_required(VERSION 3.4.3)
+
+project(QSharpPasses)
+
+find_package(LLVM REQUIRED CONFIG)
+include(CheckCXXCompilerFlag)
+
+message(STATUS "Found LLVM ${LLVM_PACKAGE_VERSION}")
+message(STATUS "Using LLVMConfig.cmake in: ${LLVM_DIR}")
+
+# Setting the standard configuration for the C++ compiler
+# Rather than allowing C++17, we restrict the ourselves to 
+# C++14 as this is the standard currently used by LLVM. While
+# there is a very small change that the difference in standard 
+# would break things, it is a possibility nonetheless.
```

Comment:
```suggestion
# Setting the standard configuration for the C++ compiler.
# Rather than allowing C++17, we restrict ourselves to 
# C++14 as this is the standard currently used by LLVM. While
# there is a very small chance that the difference in standard 
# would break things, it is a possibility nonetheless.
```

Reply:
👍